src/main/frontend/handler/editor.cljs

```diff
@@ -2848,7 +2844,7 @@
                          (not= (util/nth-safe value current-pos) "]")))
             (state/set-editor-show-page-search-hashtag! false)))
 
-        (when (and @*show-commands (not= key-code 191)) ; not /
+        (when (and @*show-commands (not= key-code 191)) ; not /   TODO: is this the .charCodeAt or the event code?
```

Comment:
Should I make this dynamic to respond to the custom trigger too? I assume so, but I wasn't sure if this was supposed to be the char code or the event code.

Reply:
Sorry for the late response, it's event's keyCode. 
https://github.com/logseq/logseq/blob/master/src/main/frontend/mixins.cljs#L101

I think we can use the event's key here instead of the keyCode so that we can compare the key with `(state/get-editor-command-trigger)`, 
https://github.com/logseq/logseq/blob/master/src/main/frontend/handler/editor.cljs#L2802